index.js

```diff
@@ -80,6 +83,13 @@ function WeatherPlusPlatform(_log, _config)
 			case "weewx":
 				this.log.info("Adding station with weather service Weewx named '" + config.nameNow + "'");
 				this.stations.push(new weewx(config.key, this.log));
+                break;
+            case "tempest":
+				this.log.info("Adding station with weather service SmartWeatherAPI named '" + config.nameNow + "'");
```

Comment:
Indentation fix (tabs vs spaces…):
```suggestion
				this.stations.push(new weewx(config.key, this.log));
				break;
			case "tempest":
				this.log.info("Adding station with weather service SmartWeatherAPI named '" + config.nameNow + "'");
```